site/class-1-load/valentijnkap/index.json

```diff
@@ -0,0 +1,7 @@
+{
+	"Dinners": {
+		"24-09-2017": "Rusty",
+		"23-09-2017": "McChicken",
+		"22-09-2017": "Moink Balls"
+	}
+}
```

Comment:
Currently, your JSON is an object.

It makes more sense to use an array for your data, such as this:

```js
[
  {
    "day": "24-09-2017",
    "dinner": "Rusty"
  },
  // ...
]
```

...and that also makes it much easier to render!